tests/integration/suite/placement/dissemination/namespaced_actors.go

```diff
@@ -0,0 +1,313 @@
+/*
```

Comment:
Rename file to `namespaced`. `actors` is implied because we are testing placement. We also want to avoid using `_` in file names.